### ISSUE TITLE
Train with the whole dataset

### ISSUE THREAD

**Issue by iuria21** [Rvbens/Chatbot-en-Espanol#1]
Hi, thanks for the repo, it's fantastic! No issues but some questions for you:

I tried to train the Transformer with the whole dataset, but the loss is getting stuck and not getting lower from +- 4.5. Do you know if i could solve it changing some parameter like the learning rate or something else?

Do you know if there is any way to force the model to answer a bit longer sentences?

thanks for your help!

**Comment by Rvbens** [comment]
Yes, this is a problem I have too when training with more than 500k lines. I currently don't know what the problem is but I want look more into it.
